catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/works/ApiWorksTest.scala

```diff
@@ -25,6 +25,11 @@ class ApiWorksTest extends ApiWorksTestBase {
                           |     "id": "${works(0).canonicalId}",
                           |     "title": "${works(0).title.get}",
                           |     "description": "${works(0).description.get}",
+                          |     "workType": {
+                          |       "id": "id",
```

Comment:
Would you object to me pushing a commit that puts more interesting strings here? Can be super annoying to debug otherwise.

Reply:
What and deprive myself of the opportunity to write them myself? No way. I need my dose of whimsy.